assets/app/views/_overview-deployment.html

```diff
@@ -49,7 +49,7 @@ <h3 class="h4">
           <span class="deployment-trigger" ng-repeat="cause in rc.causes">
             <span ng-switch="cause.type">
               <span ng-switch-when="ImageChange">
-                a new image <span ng-if="cause.imageTrigger.from">for {{cause.imageTrigger.from | imageObjectRef : null : true}}</span>
```

Comment:
@liggitt We do pass this string through the `imageObjectRef` filter, which abbreviates the identifier. I'm not sure how long these strings get.

@jhadvig You should probably roll back my suggested change here :-/